README.md

```diff
@@ -63,16 +64,15 @@ fn main() -> Result<()> {
 
 You can enable the experimental support for saving KDBX4 databases using the `save_kdbx4` feature.
 
-```rust ignore
-use anyhow::Result;
+```rust
```

Comment:
@louib why I had previously ignored this test is that otherwise, the test suite will fail if we don't run with `--all-features`.

Reply:
I addressed that in the latest commit.